avalon/pipeline.py

```diff
@@ -216,24 +216,40 @@ def remove(self, container):
 @lib.log
 class Creator(object):
     """Determine how assets are created"""
-    name = None
     label = None
     family = None
+    variants = None
 
     def __init__(self, name, asset, options=None, data=None):
-        self.name = name or self.name
```

Comment:
Removing `self.name` might introduce backwards compatibility issues - since other code could be using it. E.g. we've been using it [here](https://github.com/Colorbleed/colorbleed-config/blob/master/colorbleed/plugins/maya/create/colorbleed_renderglobals.py#L54)

How about we remove it as attribute like you did (line 219) but still assign name to `self.name` in `__init__`?

Reply:
This has been take cared in 430d364